symplyphysics/laws/relativistic/relativistic_time_dilation.py

```diff
@@ -0,0 +1,38 @@
+from sympy import Eq, solve, sqrt
```

Comment:
Remove unrelated law.

Reply:
That is from other law. I forgot to create another branch for it